test/controllers/expenses_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class ExpensesControllerTest < ActionController::TestCase
+  include Devise::Test::ControllerHelpers # For testing with Devise authentication
+
+  def setup
+    @user = FactoryBot.create(:user)
+    sign_in @user
+  end
+
+  test 'should get index' do
+    get :index
+
+    assert_response :success
+    assert_template :index
```

Comment:
- [ ] Please make sure to fix the part of your tests that failed and make all your test pass as required for this project and requested by the previous reviewers 👍 

![unnamed (2)](https://github.com/kalbek/Budget-app/assets/6615394/562fcbb2-590b-4651-bd1a-c1133f61426c)